src/status_im/subs/chats.cljs

```diff
@@ -169,23 +169,31 @@
  :chats/current-chat-chat-view
  :<- [:chats/current-chat]
  (fn [current-chat]
-   (select-keys current-chat
-                [:chat-id
-                 :able-to-send-message?
-                 :group-chat
-                 :admins
-                 :invitation-admin
-                 :public?
-                 :chat-type
-                 :color
-                 :contact-request-state
-                 :chat-name
-                 :synced-to
-                 :synced-from
-                 :community-id
-                 :emoji
-                 :description
-                 :last-message])))
+   (assoc
+    (select-keys current-chat
+                 [:chat-id
+                  :able-to-send-message?
+                  :group-chat
+                  :admins
+                  :invitation-admin
+                  :public?
+                  :chat-type
+                  :color
+                  :contact-request-state
+                  :chat-name
+                  :synced-to
+                  :synced-from
+                  :community-id
+                  :emoji
+                  :description])
+    :empty-chat?
+    (not (:last-message current-chat)))))
+
+(re-frame/reg-sub
```

Comment:
can we include unit tests to cover our subs?
cc @ilmotta

Reply:
@J-Son89, @Parveshdhull, we have this guideline already in the repo:

```
#### Subscription tests

Test [layer-3 subscriptions](https://day8.github.io/re-frame/subscriptions/) by
actually subscribing to them, so reframe's signal graph gets validated too.
```

But there's currently no mention of when a layer-3 should be tested. The easiest guideline is to always test layer-3 subs of course, but this one in this PR is actually so trivial that it's fine not to test it. Testing it would be almost like testing Clojure itself.

This sub is what I like to think as an `extractor from a layer-3 sub`, and because we don't test extractor subs (layer-2), to me it seems fine to not test extractors from layer-3 subs as well.

Does any of that make sense to you guys?

Reply:
sure, I was also wondering if the original layer-3 has tests covering it. 
Makes sense not to test the extractor subs ofc 👍